web_ir_actions_act_view_reload/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# Copyright 2018 Brainbean Apps
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+
+{
+    "name": "Web Actions View Reload",
+    "summary": "Enables reload of the current view via ActionManager",
+    "category": "Web",
+    "version": "13.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Modoolar, " "Brainbean Apps, " "Odoo Community Association (OCA)",
```

Comment:
This doesn't seem right. Does this work?

Reply:
It works because of python's auto-concatenation of strings.
Black produced this. I'll fix.

Reply:
@guewen  while we are here and i'm just wondering, how did "Brainbean Apps" become author of this module? If i recall correctly they just migrated it to 12.0 without adding any new functionality. Is there any OCA rule about this?

Reply:
The effort of migrating a module can be big depending the version. There's no strict rule for saying if someone work grants the co-authorship. @alexey-pelykh can you tell us arguments for you to be there?

Reply:
@pedrobaeza this module is quite heavy for migration, v12 was nearly a rewrite if I remember correctly

Reply:
OK, if that was the case, then I think it's reasonable for you to be in the author list

Reply:
https://github.com/OCA/web/commit/633e0c8a2d3e8e2a7d9cda18b1b2ee5c98481450#diff-b3ac0207ff2950bb6f972a8d4de64810 not much remained

Reply:
I even did a mistake with copyright :D

Reply:
@pedrobaeza @alexey-pelykh guys, thanks for clarification :)